checksums.sh

```diff
@@ -15,18 +15,19 @@ then
 fi
 
 SUBJECT_FOLDER=$1
+QSUB_FILE=".qsub.out.*"
 
-echo "###########################################"
-echo "############ Checksum of docker image #####"
-echo "###########################################"
-
-grep "Digest: sha256:" .qsub.out.*
+count=`ls -a 2>/dev/null | grep $QSUB_FILE | wc -l`
+if [ $count != 0 ]
+then
```

Comment:
We should stop as you said. Even if the image is not getting pulled , the sha digest in shown in the .qsub.out files.

> CBRAIN Task Starting At 1483719502 : 2017-01-06 11:18:22
docker pull bigdatalabteam/hcp-prefreesurfer:exec-centos5-fslbuild-centos5-latest
exec-centos5-fslbuild-centos5-latest: Pulling from bigdatalabteam/hcp-prefreesurfer
Digest: sha256:f0320a72501b0b8cf25ebaec11801a7d1d7e55857fcb699b57aa433d0d6d65e9
Status: Image is up to date for bigdatalabteam/hcp-prefreesurfer:exec-centos5-fslbuild-centos5-latest

So yes we should stop execution.